### ISSUE TITLE
Pen calibration SP4 Wayland

### ISSUE THREAD

**Issue by afalout** [linux-surface/iptsd#36]
Hello, pen works, including pressure. That good. Thanks!
Not so good is that the pen draws few inches up and right from where pen is. Tested in several apps including Xournal++, Write & OneNote running on Anbox
While Wacom folks clearly are watching Wayland:
https://developer-docs.wacom.com/intuos-cintiq-business-tablets/docs/wayland-faqs
https://developer-docs.wacom.com/intuos-cintiq-business-tablets/docs/wayland-overview
... ATM, Wacom tools are not compatible with Wayland, so cant use that for calibration.
Is there a way to calibrate pen in this situation? 
Thanks!

**Comment by afalout** [comment]
So I'm the only one wanting to use pen on Linux with Wayland ? :(

**Comment by StollD** [comment]
No, but you might be the only one that needs to calibrate the Pen.

There are no tools to calibrate the Pen, except for the calibrator that is built into GNOME (Wacom tools won't work since it is not a Wacom pen / digitizer).

**Comment by afalout** [comment]
Thanks for your response! So are you saying that pen works correctly for other people, so they dont need to calibrate it?

Im on KDE/Wayland, and libwacom does not support Wayland. Can anyone confirm is there pen works correctly in this configuration?

BTW - libwacom fork for Intel Precise Touch and Stylus does exist:
        https://github.com/linux-surface/libwacom 

Thanks,
Andrej

**Comment by StollD** [comment]
Libwacom is just a database of device IDs and a library to query them. DEs can use it to list all wacom-like devices but the only DE that actually does that is GNOME. It doesnt impact the functionality of the device.

I used KDE with Wayland for a while in the past and everything was working fine. I would assume that it works fine for most people since I havent seen reports like this (or only a few).

Did you check if the pen works properly under Xorg? Are the GTK apps you mentioned (Xournal++) running using Wayland (run `xkill` or `xprop` and see if you can select them. If yes they use Xorg not Wayland)

**Comment by averil-aussedat** [comment]
If anyone looks into calibration of a Wacom tablet with Wayland, here is a life-saving tutorial to create a udev rule: https://support.displax.com/portal/en/kb/articles/orientation-using-udev#Configure_Touchscreen_Orientation_in_Linux_Using_udev_Rules
